extensions/wikia/DesignSystem/ds.built.js

```diff
@@ -0,0 +1,768 @@
+			// get current location
+			var location = typeof window !== "undefined" && window.location;
+
+			if (!location) {
+				throw new Error("fixUrls requires window.location");
+			}
+
+			// blank or null?
+			if (!css || typeof css !== "string") {
+				return css;
+			}
+
+			var baseUrl = location.protocol + "//" + location.host;
+			var currentDir = baseUrl + location.pathname.replace(/\/[^\/]*$/, "/");
+
+			// convert each url(...)
+			/*
+			 This regular expression is just a way to recursively match brackets within
+			 a string.
+
+			 /url\s*\(  = Match on the word "url" with any whitespace after it and then a parens
+			 (  = Start a capturing group
+			 (?:  = Start a non-capturing group
+			 [^)(]  = Match anything that isn't a parentheses
+			 |  = OR
+			 \(  = Match a start parentheses
+			 (?:  = Start another non-capturing groups
+			 [^)(]+  = Match anything that isn't a parentheses
+			 |  = OR
+			 \(  = Match a start parentheses
+			 [^)(]*  = Match anything that isn't a parentheses
+			 \)  = Match a end parentheses
+			 )  = End Group
+			 *\) = Match anything and then a close parens
+			 )  = Close non-capturing group
+			 *  = Match anything
+			 )  = Close capturing group
+			 \)  = Match a close parens
+
+			 /gi  = Get all matches, not the first.  Be case insensitive.
+			 */
+			var fixedCss = css.replace(/url\s*\(((?:[^)(]|\((?:[^)(]+|\([^)(]*\))*\))*)\)/gi, function(fullMatch, origUrl) {
+				// strip quotes (if they exist)
+				var unquotedOrigUrl = origUrl
+					.trim()
+					.replace(/^"(.*)"$/, function(o, $1){ return $1; })
+					.replace(/^'(.*)'$/, function(o, $1){ return $1; });
+
+				// already a full url? no change
+				if (/^(#|data:|http:\/\/|https:\/\/|file:\/\/\/)/i.test(unquotedOrigUrl)) {
+					return fullMatch;
+				}
+
+				// convert the url to a full url
+				var newUrl;
+
+				if (unquotedOrigUrl.indexOf("//") === 0) {
+					//TODO: should we add protocol?
+					newUrl = unquotedOrigUrl;
+				} else if (unquotedOrigUrl.indexOf("/") === 0) {
+					// path should be relative to the base url
+					newUrl = baseUrl + unquotedOrigUrl; // already starts with '/'
+				} else {
+					// path should be relative to current directory
+					newUrl = currentDir + unquotedOrigUrl.replace(/^\.\//, ""); // Strip leading './'
+				}
+
+				// send back the fixed url(...)
+				return "url(" + JSON.stringify(newUrl) + ")";
+			});
+
+			// send back the fixed css
+			return fixedCss;
+		};
+
+
+		/***/ }),
+	/* 9 */
+	/***/ (function(module, exports, __webpack_require__) {
+
+		// style-loader: Adds some css to the DOM by adding a <style> tag
+
+		// load the styles
+		var content = __webpack_require__(5);
+		if(typeof content === 'string') content = [[module.i, content, '']];
+		// add the styles to the DOM
+		var update = __webpack_require__(7)(content, {"convertToAbsoluteUrls":false,"sourceMap":false});
+		if(content.locals) module.exports = content.locals;
+		// Hot Module Replacement
+		if(false) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Change this condition so that it does not always evaluate to "false". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2583)